src/index.js

```diff
@@ -1,17 +1,28 @@
 import { loaded } from 'document-promises';
 
 customElements.define('iframe-defer', class extends HTMLElement {
+  constructor() {
+    super();
+    this._iframe = document.createElement('iframe');
+  }
+
+  static get observedAttributes() {
+    return ['width', 'height'];
```

Comment:
I think we should not hardcode this. Something like `return Object.keys(HTMLIFrameElement.prototype)` will do.